intermine_boot/intermine_docker.py

```diff
@@ -243,7 +243,9 @@ def create_intermine_builder_container(client, image, env, options):
         'MINE_REPO_URL': os.environ.get('MINE_REPO_URL', ''),
         'MEM_OPTS': os.environ.get('MEM_OPTS', '-Xmx2g -Xms1g'),
         'IM_DATA_DIR': os.environ.get('IM_DATA_DIR', ''),
-        'FORCE_MINE_BUILD': 'true' if options['datapath_im'] else 'false'
+        'FORCE_MINE_BUILD': 'true' if (
+               options['datapath_im'] and not _is_conf_same(env['data_dir', options])
```

Comment:
It seems the `]` is misplaced here, should be `_is_conf_same(env['data_dir'], options)`.